test/cli/require-qunit-test.js

```diff
@@ -0,0 +1,59 @@
+const proxyquire = require( "proxyquire" );
+const resolveStub = {
+	sync( path ) {
+		return path;
+	}
+};
+
+QUnit.module( "requireQUnit", function() {
+	QUnit.test( "finds QUnit in the current working directory", function( assert ) {
+		const localQUnit = {
```

Comment:
We can do that if you feel it is really needed. I opted for the current format so that the variables had more detailed names within each test.